tests/parsing_data/test_fetch_data.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from python_dwd import collect_dwd_data
+from python_dwd.enumerations.parameter_enumeration import Parameter
+from python_dwd.enumerations.period_type_enumeration import PeriodType
+from python_dwd.enumerations.time_resolution_enumeration import TimeResolution
+
+
+@pytest.mark.remote
+def test_fetch_and_parse_dwd_data():
```

Comment:
Shouldn't this test be in the "test_data_collection" file? Also what happens if station "1048" is not available anymore? Rather loop over large range and take first station that is available. Also "collect_dwd_data" relies on the filelist that is previously created, so for the test make sure to use create_new_filelist=True to get a "fresh" filelist.

Reply:
How could I have missed that one? Thanks for pointing out the right module to put this in.